### PR TITLE
Adding SAUDIA logo

### LINKED ISSUE #9158: Saudia
### Brand Name

Saudia

### Website

https://www.saudia.com/

### Popularity Metric

Global Ranking: 7,327   https://www.similarweb.com/website/saudia.com/#overview

### Official Resources for Icon and Color

unknown

### Additional Comments

unknown

### Contributing

No

### PR DESCRIPTION
**Pull request by CodeWithShipraJain**
**Issue:** closes #9158

**Similarweb rank:** 7,327 https://www.similarweb.com/website/saudia.com/#overview
  

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

**Review by adamrusted — APPROVED** [review]
I'm happy with this for a manual vectorization, thanks @CodeWithShipraJain 🎉
Will wait for another @simple-icons/icon-review member to also confirm before we merge though.
image

**Comment by PeterShaggyNoble** [comment]
Saudia have since [updated their branding](https://www.saudia.com/pages/experience/about-us/corporate-communication/press-releases-and-news/press-releases/press-release-30092023). Would you mind having another go at this, please, @CodeWithShipraJain? I'm on the hunt for a vector version of their logo to make things easier; I'll let you know if I find one.

**Comment by PeterShaggyNoble** [comment]
Found some vector artwork at https://www.saudiatechnic.com/media-center/press-kit

**Review by adamrusted — CHANGES_REQUESTED** [review]
See https://github.com/simple-icons/simple-icons/pull/9178#issuecomment-1820775637

File: _data/simple-icons.json
Comment (PeterShaggyNoble) [inline _data/simple-icons.json:10981]:
```suggestion
            "title": "Saudia",
            "hex": "026938",
            "source": "https://www.saudiatechnic.com/media-center/press-kit"
```

File: icons/saudia.svg
Comment (PeterShaggyNoble) [inline icons/saudia.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Saudia</title><path d="M12 3.226c-.016.03-.208.378-.248.542a2.915 2.915 0 0 0-.058.301 1.364 1.364 0 0 0-.42-.256c-.34-.128-.732-.03-.732-.03s.183.089.346.239c.07.063.123.138.166.223-.53-.08-1.205-.076-1.896.229-.93.41-1.494 1.3-1.494 1.3s.63-.49 1.577-.71a3.525 3.525 0 0 1 1.268-.056c-.593.13-1.422.439-2.034 1.218-.533.678-.668 1.627-.668 1.627s.432-.554 1.16-1.172a5.346 5.346 0 0 1 1.208-.774c-.454.43-.678 1.09-.768 1.341-.277.763-.1 1.612-.1 1.612s.117-.555.542-1.275a4.474 4.474 0 0 1 1.019-1.17c-.39 1.448.012 2.874.012 2.874s.245-1.098.613-2.028A3.78 3.78 0 0 1 12 6.34c.172.225.351.524.508.92.368.93.612 2.028.612 2.028s.404-1.426.013-2.875c.326.268.717.657 1.02 1.17.424.72.542 1.276.542 1.276s.175-.85-.1-1.612c-.092-.25-.316-.91-.77-1.34.357.165.78.409 1.209.773a8.883 8.883 0 0 1 1.16 1.172s-.135-.949-.668-1.627c-.612-.78-1.442-1.089-2.034-1.218.37-.049.801-.053 1.268.056.946.22 1.577.71 1.577.71s-.565-.89-1.494-1.3c-.69-.305-1.366-.309-1.896-.229a.758.758 0 0 1 .166-.223c.163-.15.346-.239.346-.239s-.392-.098-.733.03a1.364 1.364 0 0 0-.42.256 2.873 2.873 0 0 0-.057-.301c-.04-.164-.232-.513-.249-.542ZM.026 7.916l3.538 3.91c1.082-.192 2.175-.2 3.249.035 1.608.352 3.106 1.062 4.58 1.777l-.136-3.643L.114 7.763c-.064-.025-.16.073-.088.153zm3.538 3.91 4.042 4.469a4.296 4.296 0 0 1-.376.246c-1.34.793-2.152.904-3.24.984 1.002.21 1.426.482 2.893.15.516-.115 1.343-.646 1.371-.664l3.405 3.763-.214-5.76a137.295 137.295 0 0 0-3.193 1.992l-.638-.709c.431-.238.861-.479 1.288-.725.715-.413 1.415-.852 2.132-1.263-1.71-.979-2.966-1.566-4.025-1.929-1.09-.43-2.285-.514-3.445-.555Zm20.323-4.063L12.744 9.995l-.136 3.643c1.473-.715 2.974-1.425 4.582-1.777 1.074-.235 2.165-.227 3.247-.036l3.538-3.909a.092.092 0 0 0-.088-.153zm-3.45 4.062c-1.16.04-2.356.126-3.445.555-1.236.424-2.487 1.055-4.025 1.929.716.411 1.417.85 2.132 1.263.427.246.857.487 1.288.725l-.638.709c-2.007-1.277-2.528-1.592-3.194-1.992l-.213 5.76 3.405-3.763c.028.018.855.549 1.37.665 1.468.33 1.892.058 2.893-.15-1.088-.081-1.9-.192-3.239-.985a4.297 4.297 0 0 1-.377-.246z"/></svg>
```

**Review by PeterShaggyNoble — CHANGES_REQUESTED** [review]
Saudia

Updated with the new branding, extracting the icon from `Saudia Technic_Stacked Arabic_MonoSaudia Technic_Stacked Arabic` in the bottom left of the [Arabic Saudia Technic Logo Suite PDF](https://www.saudiatechnic.com/upload/assets/Saudia_Technic_logo_suite_Arabic.pdf) found at [source](https://www.saudiatechnic.com/media-center/press-kit) and taking the colour from [the SVG](https://www.saudiatechnic.com/upload/general/logo-dark.svg) in the header.

**Review by adamrusted — COMMENTED** [review]
@PeterShaggyNoble I'm seeing a difference between source in red and your PR in black.
image

Knowing our history of differences, can another @simple-icons/icon-review bod validate one way or the other?

**Comment by PeterShaggyNoble** [comment]
I think this one might be a clash of software again, @adamrusted, I'm not seeing that at all in Inkscape.

**Comment by service-paradis** [comment]
I'm seeing a hairline difference over one of the right leaf:
image

I dont see the difference mentioned by @adamrusted, so I think it should be good to go!

**Comment by PeterShaggyNoble** [comment]
That should be fixed now, @service-paradis.

**Review by service-paradis — APPROVED** [review]
LGTM now, thank @PeterShaggyNoble!

**Pull request merged**
